metrics/sink_test.go

```diff
@@ -68,7 +69,6 @@ func TestGaugeSink(t *testing.T) {
 	})
 	t.Run("calc", func(t *testing.T) {
 		sink := GaugeSink{}
-		sink.Calc()
```

Comment:
It seems to me like the whole test run (`t.Run("calc", func(t *testing.T)`) here (and for other metric types) is no longer needed, should we clean up them since it seems like it's the best moment to do that?

Reply:
Good catch 👀 I missed those 🙇🏻